src/components/atoms/Box.tsx

```diff
@@ -0,0 +1,9 @@
+import { memo } from 'react'
+import { View, ViewProps } from 'react-native'
+
+import { generateStyledComponent } from '../utils/generateStyledComponent'
+import { StyledProps } from './types'
+
+export type BoxProps = StyledProps & ViewProps
+
+export const Box = memo(generateStyledComponent<BoxProps>(View))
```

Comment:
I'm thinking of putting forwardRef as well

Reply:
@MSzalowski I will create separate issue for that. I think the best solution would be to add forward ref to the generateStyledComponent function (I don't know if it's possible 🤔 )